var/spack/repos/builtin/packages/julia/package.py

```diff
@@ -3,206 +3,184 @@
-    variant('mkl', default=False, description='Use Intel MKL')
-
-    patch('gc.patch', when='@0.4:0.4.5')
-    patch('openblas.patch', when='@0.4:0.4.5')
-    patch('armgcc.patch', when='@1.0.0:1.1.1 %gcc@:5.9 target=aarch64:')
-
-    # Build-time dependencies:
-    # depends_on('awk')
-    depends_on('m4', type='build')
-    # depends_on('pkgconfig')
-    # Python only needed to build LLVM?
-    depends_on('python@2.7:2.8', type='build', when='@:1.1')
-    depends_on('python@2.7:', type='build', when='@1.2:')
-    depends_on('cmake@2.8:', type='build', when='@1.0:')
-    depends_on('cmake@:3.11', type='build', when='@:1.4')
-    depends_on('git', type='build', when='@master')
-
-    # Combined build-time and run-time dependencies:
-    # (Yes, these are run-time dependencies used by Julia's package manager.)
-    depends_on('cmake @2.8:', type=('build', 'run'), when='@:0.6')
-    depends_on('curl', when='@:0.5.0')
-    depends_on('git', type=('build', 'run'), when='@:0.4')
-    depends_on('openssl@:1.0', when='@:0.5.0')
-    depends_on('mkl', when='+mkl')
-
-    # Run-time dependencies:
-    # depends_on('arpack')
-    # depends_on('fftw +float')
-    # depends_on('gmp')
-    # depends_on('libgit')
-    # depends_on('mpfr')
-    # depends_on('openblas')
-    # depends_on('pcre2')
-
-    # ARPACK: Requires BLAS and LAPACK; needs to use the same version
-    # as Julia.
-
-    # BLAS and LAPACK: Julia prefers 64-bit versions on 64-bit
-    # systems. OpenBLAS has an option for this; make it available as
-    # variant.
-
-    # FFTW: Something doesn't work when using a pre-installed FFTW
-    # library; need to investigate.
-
-    # GMP, MPFR: Something doesn't work when using a pre-installed
-    # FFTW library; need to investigate.
-
-    # LLVM: Julia works only with specific versions, and might require
-    # patches. Thus we let Julia install its own LLVM.
-
-    # Other possible dependencies:
-    # USE_SYSTEM_OPENLIBM=0
-    # USE_SYSTEM_OPENSPECFUN=0
-    # USE_SYSTEM_DSFMT=0
-    # USE_SYSTEM_SUITESPARSE=0
-    # USE_SYSTEM_UTF8PROC=0
-    # USE_SYSTEM_LIBGIT2=0
-
-    conflicts('+cxx', when='@:0.6', msg='Variant cxx requires Julia >= 1.0.0')
-
-    conflicts('@:0.7.0', when='target=aarch64:')
-
-    # GCC conflicts
-    conflicts('@:0.5.1', when='%gcc@8:', msg='Julia <= 0.5.1 needs GCC <= 7')
-
-    # Building recent versions of Julia with Intel is untested and unsupported
-    # by the Julia project, https://github.com/JuliaLang/julia/issues/23407.
-    conflicts('@0.6:', when='%intel',
-              msg='Only Julia <= 0.5.x can be built with the Intel compiler.')
-    conflicts('%intel', when='~mkl',
-              msg='Building with the Intel compiler requires the mkl variant '
-              '(+mkl)')
+    version('1.7.0-rc3', sha256='87604d53bc0832045522bc92ee98923fca370ead43f567c0676c6b9ecf1c5134')
+
+    # We've deprecated these versions, so that we can remove them in Spack 0.18
```

Comment:
We might not want to deprecate 1.6 since that will be the LTS.

Reply:
It would sure be easier if 1.7 was the LTS :p

Reply:
Yeah I think we lost that argument ;)

Reply:
1.6.4 now works too :upside_down_face: